### ISSUE TITLE
make annotation comments a bit more useful

### ISSUE THREAD

**Issue by mah11** [pombase/canto#979]
At present comments on annotations are kind of a dead end -- they get made and then just sit in the canto sessions, where someone may or may not ever read them.

We agreed that there's no current need for the comments to go public (e.g. on gene pages), but it would be nice to do a bit more with them than we do now.

Kim - I think you said they can be put into Chado? If so, let's do it.

Also add something that allows some simple comment searching. Just text matching would be plenty. I tend to use comments for things like exogenous assay substrates or other things that don't fit neatly into prefix:ID extensions.

Maybe somewhere in the admin interface?

And I think I'm also going to open another pie-in-the-sky ticket for some other session-searching ideas that have just popped into my head ...

**Comment by kimrutherford** [comment]
> Kim - I think you said they can be put into Chado? If so, let's do it.

Yep, there's no problem storing them in Chado.  My only concern is that they are comments to the curators so perhaps it's not appropriate to put them in Chado?  We put the Chado dump file on the ftp site.  I suspect no one has ever downloaded it but technically it's public so the comments to the curators will be public too.

> Also add something that allows some simple comment searching.

The quickest thing to implement would be a page that has the comments and links to the session in one big table.  Then you could just do a text search in your browser.

**Comment by mah11** [comment]
> My only concern ...

I can't recall adding any specific comment that I would need to hide from the world. Mostly I use comments for experiment details that don't fit nicely into extensions, conditions etc., e.g. "they used bulk calf thymus histones in the kinase assay", so (a) they're not really "only" for curators and (b) the major risk associated with having them in the chado dumps is that anyone who reads them might be bored to death.

> The quickest thing to implement would be a page ...

Quick is good. Simple is good. I don't envision using a comment search often enough to make it worth spending time coding something not-simple.
